packages/features/form-builder/FormBuilderField.tsx

```diff
@@ -227,7 +244,7 @@ export const ComponentForField = ({
   }
   if (componentConfig.propsType === "text") {
     return (
-      <WithLabel field={field} readOnly={readOnly}>
+      <WithLabel field={field} readOnly={readOnly} noLabel={noLabel}>
```

Comment:
Pass on `noLabel` directly instead of setting field.label to empty string because that would mean that someone wanting to use the field.label still won't have it.